src/components/modals/TradePreviewModalGP.vue

```diff
@@ -1,70 +1,189 @@
 <template>
-  <BalModal
-    :show="open"
-    @close="onClose"
-    :title="$t('previewTradeTransactions')"
-  >
+  <BalModal show @close="onClose" :title="$t('previewTrade')">
     <div>
-      <div
-        class="-mx-4 p-4 flex items-center border-b border-t dark:border-gray-700"
-      >
-        <BalAssetSet
-          :addresses="[addressIn, addressOut]"
-          :size="30"
-          :width="55"
-        />
-        <div class="flex flex-col">
-          <div class="font-bold">
-            {{ fNum(amountIn, 'token') }} {{ symbolIn }} ->
-            {{ fNum(amountOut, 'token') }} {{ symbolOut }}
+      <BalCard noPad class="relative mb-6">
+        <template v-slot:header>
+          <div class="w-full p-3 border-b bg-gray-100 rounded-t-lg text-sm">
+            {{ $t('effectivePrice') }}
+            {{
+              trading.exactIn.value
+                ? trading.effectivePriceMessage.value.tokenIn
+                : trading.effectivePriceMessage.value.tokenOut
+            }}
+          </div>
```

Comment:
I feel like this effective price header could be a bit more subtle and the label could use a bit of visual distinction from the amounts. From this:
<img width="537" alt="Screenshot 2021-07-05 at 19 11 32" src="https://user-images.githubusercontent.com/2406506/124507115-00675580-ddc5-11eb-9356-271de07e3b97.png">
to this:
<img width="578" alt="Screenshot 2021-07-05 at 19 11 14" src="https://user-images.githubusercontent.com/2406506/124507128-0a895400-ddc5-11eb-9323-0fa7ac1ec221.png">
```suggestion
      <BalCard noPad class="relative mb-6">
        <template v-slot:header>
          <div class="w-full p-3 border-b bg-gray-50 rounded-t-lg text-xs">
            <span class="text-gray-500">
              {{ $t('effectivePrice') }}
            </span>
            {{
              trading.exactIn.value
                ? trading.effectivePriceMessage.value.tokenIn
                : trading.effectivePriceMessage.value.tokenOut
            }}
          </div>
```